### PR TITLE
[Backport][ipa-4-6] Normalize test definations titles

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
Manual Backport of #4151

Signed-off-by: Gaurav Talreja <gtalreja@redhat.com>

### CODE REVIEW

File: ipatests/prci_definitions/gating.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
         timeout: 3600
         topology: *master_1repl
 
-  fedora-27/external_ca:
+  fedora-27/test_external_ca_TestExternalCA:
```
Comment (flo-renaud) [inline ipatests/prci_definitions/gating.yaml]: To keep a consistent name between the branches, I would rather use the name `fedora27/external_ca_1`.

File: ipatests/prci_definitions/nightly_ipa-4-6.yaml [hunk 2]
```diff
@@ -59,7 +59,7 @@ jobs:
         timeout: 3600
         topology: *master_1repl
 
-  fedora-27/external_ca_1:
+  fedora-27/test_external_ca_TestExternalCA:
```
Comment (flo-renaud) [inline ipatests/prci_definitions/nightly_ipa-4-6.yaml]: I would rather keep the name external_ca_1 to be consistent between the branches.

File: ipatests/prci_definitions/nightly_ipa-4-6.yaml [hunk 3]
```diff
@@ -71,7 +71,7 @@ jobs:
         timeout: 4800
         topology: *master_1repl_1client
 
-  fedora-27/external_ca_2:
+  fedora-27/test_external_ca_TestSelfExternalSelf:
```
Comment (flo-renaud) [inline ipatests/prci_definitions/nightly_ipa-4-6.yaml]: Same comment, I would rather keep the name external_ca_2.

File: ipatests/prci_definitions/gating.yaml (hunk 1 above)
Comment (Gauravtalreja1) [reply ipatests/prci_definitions/gating.yaml]: @flo-renaud here external_ca titles make bit more sense rather than using external_ca1 and external_ca2.

**Comment by flo-renaud** [comment]
Hi @Gauravtalreja1 
LGTM but could you update the commit msg?

**Comment by flo-renaud** [comment]
ipa-4-6:

* 636ea489bb59ed0b26951299053db5651c78a20f Normalize test definations titles

**Pull request closed without merging**